web/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,248 @@
+
+  function handleChange(value: string) {
+    setText(value);
+  }
+
+  const user_views = 1;
+  const user_saves = mockData.length;
+  const user_name = "username";
+  const user_icon = "https://avatars.githubusercontent.com/u/79925808?v=4";
+
+  return (
+    <>
+      {/* TOP BORDER */}
+      <div className="p-4"></div>
+
+      {/* PROFILE PICTURE */}
+      <div className="container mx-auto">
+        {/* START PHOTO PADDING */}
+        <div className="p-4"></div>
+        {/* FLEX-SEAL (wrap*) */}
+        <div className="flex flex-wrap items-center justify-center">
+          <a href="#" className="relative block">
+            {/* PROFILE PICTURE DATA */}
+            <img
+              className="custom-position h-24 w-24 rounded-full object-cover"
+              src={user_icon}
+              alt="name"
+            />
+          </a>
+        </div>
+        {/* END PHOTO PADDING */}
+        <div className="p-4"></div>
+      </div>
+
+      {/* USERNAME */}
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-center">
+          <div className="text-lg">@{user_name}</div>
+        </div>
+        <div className="p-1"></div>
+      </div>
+
+      {/* VIEWS / SAVED */}
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-center gap-2">
+          <div className="text-sm">{user_views} views</div>
+          <div className="text-sm">{user_saves} saves</div>
+        </div>
+        <div className="p-2"></div>
+      </div>
+
+      {/* CREATED / COLLECTIONS */}
+      <div className="container mx-auto">
+        <div className="p-2"></div>
+        <div className="flex flex-wrap items-center justify-center gap-2">
+          <div className="p-2">
+            <section className={"flex gap-4"}>
+              <div className={"flex flex-col items-center gap-2"}>
+                {/*<Button disabled>Created</Button>*/}
+                <Button variant={"text"}>Created</Button>
+              </div>
+              {/*<div className={"flex flex-col items-center gap-2"}>
+                <Button>Collections</Button>
+              </div>*/}
+              <Link href={"/Profile/collections"}>
+                <div className={"flex flex-col items-center gap-2"}>
+                  <Button>Collections</Button>
+                </div>
+              </Link>
+            </section>
+          </div>
+        </div>
+        <div className="p-2"></div>
+      </div>
+
+      {/* SEARCH / +CREATE */}
+      <div className="container mx-auto">
+        <div className="flex flex-wrap items-center justify-center gap-4">
+          <SearchBar
+            value={text}
+            onValueChange={handleChange}
+            className={"w-11/12 max-w-xl sm:w-2/3"}
+          />
+          <LinkIconButton
+            href={"/create"}
+            className={"hidden text-base sm:block"}
+          >
+            {" "}
```

Comment:
Usually, using spacing for layout is a bad idea. I would suggest just adding some left margin to the `<MdAddCircle />`